pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2839,7 +2839,7 @@ type DisruptionManagementSpec struct {
 	// +optional
 	OSDMaintenanceTimeout time.Duration `json:"osdMaintenanceTimeout,omitempty"`
 
-	// PGHealthCheckTimeout is the time (in minutes) that the operator will wait for the placement groups to become
+	// DEPRECATED: PGHealthCheckTimeout is the time (in minutes) that the operator will wait for the placement groups to become
```

Comment:
How about replace this setting description completely? (and delete the next 3 lines too)
```suggestion
	// DEPRECATED: PGHealthCheckTimeout is no longer implemented
```

Reply:
done.